enocean/plugin.yaml

```diff
@@ -2,23 +2,93 @@
 plugin:
     # Global plugin attributes
     type: gateway                 # plugin type (gateway, interface, protocol, system, web)
-    description:                  # Alternative: description in multiple languages
-        de: 'Enocean Unterstützung'
-        en: ''
+    description:
+        # Alternative: description in multiple languages
+        de: 'Plugin für Enocean Unterstützung'
+        en: 'Plugin for enocean support'
     maintainer: '? (Robert Budde / aschwith)'
-#    tester:                        # Who tests this plugin?
+    # Who tests this plugin?
+    tester: '? (Robert Budde / aschwith)'
     keywords: enocean
-#    documentation: https://github.com/smarthomeNG/smarthome/wiki/CLI-Plugin        # url of documentation (wiki) page
+    # url of documentation (wiki) page
+    documentation: https://github.com/smarthomeNG/plugins/tree/master/enocean
+    
 
     version: 1.3.3                 # Plugin version
     sh_minversion: 1.3             # minimum shNG version to use this plugin
-#    sh_maxversion:                 # maximum shNG version to use this plugin (leave empty if latest)
+    sh_maxversion:                 # maximum shNG version to use this plugin (leave empty if latest)
     multi_instance: False          # plugin supports multi instance
     classname: EnOcean             # class containing the plugin
 
 parameters:
     # Definition of parameters to be configured in etc/plugin.yaml
-    
+    serialport:
+        type: str
+        default: /dev/ttyAMA0
+        description:
+            de: 'Name der Schnittstelle an der sich der Enocean Adapter befindet'
+            en: 'name of the port where the enocean adapter is plugged'
+        valid_list:
+          - /dev/ttyAMA0
+          - /dev/ttyS0
+          - /dev/ttyS1
+          - ...
```

Comment:
AFAIK, the `valid_list` contains possible values for this parameter. In this case you will only be allowed to specify the three mentioned devices and `...`.

And I think listing devices there is not a good idea since the names of the devices could be very different (depending on the system and the hardware you're using).

Maybe we could introduce a pattern support for `valid_list` setting which can be used to specify a regex pattern which will be matched against the values. But this is something @msinn can decide if this is a good or bad idea.